app/src/ui/lib/list/section-list.tsx

```diff
@@ -1383,9 +1383,14 @@ export class SectionList extends React.Component<
   }
 
   private get totalHeight() {
+    // Exit early if no rows present
+    if (this.props.rowCount.length === 0) {
+      return 0
+    }
+
```

Comment:
```suggestion
```

I think I would remove these lines and leave only the default value in `reduce`: it's its purpose and keeps the code simpler 😄

Reply:
Did not think about it that way, but sure :-D

I guess I wanted to have some documentation of the purpose, but that's fine.

Reply:
And that makes total sense! But being `reduce` a well-known API, I think having `0` as default/initial value should be enough.